webapp/src/main/java/com/box/l10n/mojito/rest/scheduledjob/ScheduledJobWS.java

```diff
@@ -0,0 +1,129 @@
+package com.box.l10n.mojito.rest.scheduledjob;
+
+import com.box.l10n.mojito.entity.ScheduledJob;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobDTO;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobManager;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobRepository;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobResponse;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobKey;
+import org.quartz.SchedulerException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+/**
+ * Webservice for enabling/disabling and triggering scheduled jobs.
+ *
+ * @author mattwilshire
+ */
+@RestController
+@ConditionalOnProperty(value = "l10n.scheduledJobs.enabled", havingValue = "true")
+public class ScheduledJobWS {
```

Comment:
I think it would be handy to have an endpoint which gets all the jobs by status? So you can see all the running jobs, or all the failed jobs etc

Reply:
Maybe a date filter too if we need to debug something and we don't want walls / pages of text

Reply:
Noted, this would be something to add for v2.